src/main/resources/views/CommercialVehicles/VTG30Welsh.hbs

```diff
@@ -84,7 +84,7 @@
             <div class="header__left">
                 <h1 class="header__title">
                     {{#if reissue}}
-                    <p class="heading-info--no-margin">{{reissue.reason}}</p>
+                    <p class="heading-info--no-margin">Tystysgrif</p>
```

Comment:
Sometimes Welsh reverses the order of the words. I believe it's actually 'Amnewid' for 'Replacement', i've just done a google translate and checked the PDF attached to the ticket.

Reply:
'Tystysgrif' is certificate I believe.